components/articles/EditArticleForm/EditLocaleForm.js

```diff
@@ -92,6 +91,9 @@ const EditLocaleForm = ({ article, prefix, formApi, onRemove, pending, errors })
           formApi={formApi}
           id={`${prefix}.slug`}
           field={`${prefix}.slug`}
+          onChange={slugValue =>
+            formApi.setValue(`${prefix}.slug`, replaceSpaceAndUnderscoreToDash(slugValue))
+          }
```

Comment:
what about more general `replaceToDash([' _'], slugValue)`?

Reply:
Nice

Reply:
or, mb just `formatSlug` and also and slashes replacement

Reply:
@Drapegnik Are you sure, that we will use this func only for slug in the future?